### ISSUE TITLE
Mapping MIDI Knobs to cue volume

### ISSUE THREAD

**Issue by StoneLabs** [FrancescoCeruti/linux-show-player#99]
Hey,
hust a small freature request. It would be great to have the ability to map MIDI knobs to cue volumes. Since most MIDI interfaces have such knobs, why not give the ability to use them?
Great project btw!

**Comment by FrancescoCeruti** [comment]
Thanks :)
This require some work, the general idea is to allow to control multiple parameter of cue, using different protocols (MIDI/OSC/...)

**Comment by StoneLabs** [comment]
I agree, making every parameter midi/osc controllabe would be awesome!
